templates/_linux.yaml

```diff
@@ -34,6 +34,7 @@ objects:
     name: ${NAME}
     labels:
       vm.kubevirt.io/template: {{ os }}-{{ item.workload }}-{{ item.flavor }}
+      vm.kubevirt.io/template.version: "{{ lookup('env', 'VERSION') | regex_replace('^v?(?P<version>\d+\.\d+\.\d+)$', '\\g<version>') }}"
```

Comment:
There can only be one template with the same name so maybe we should expose the version in the template name directly. In other words, why not create NVR naming scheme?

Reply:
good idea, but not sure how's gonna play out with non-tag versions. Let's say we do scratch builds. What `version` should be?
Furthermore, can we ever have Revision != 1 ?

Reply:
Upstream we will definitely always have revision = 1. For scratch builds.. I need to think about this, because I do not want to install new set of templates every time a developer does a commit. Maybe we could encode NV in name and encode the R in annotation just for reference purposes.

Reply:
changing the `name` to `name-version` and encoding the revision into a annotation sounds good to me.
However, this will make upgrade from last released version interesting. We will effectively have duplicate templates, not sure how we can handle that

Reply:
That is the idea. I wanted to allow something like compatibility levels using template versions. Hmm to be able to search for the right one we need to add a label with version too.

Reply:
ack, got it